src/common/schemas/specifications/suspended-payment-cancellation.json

```diff
@@ -9,11 +9,11 @@
       "$ref": "address",
       "description": "The address of the owner of the suspended payment to cancel."
     },
-    "paymentSequence": {
+    "suspensionSequence": {
```

Comment:
Maybe `suspensionTxSequence`? 
For someone who new to ripple it is now obvious that sequence refers to transaction.